src/conn/conn_compact.c

```diff
@@ -144,7 +401,7 @@ __compact_server(void *arg)
             continue;
 
         /* Find the next URI to compact. */
-        WT_ERR_NOTFOUND_OK(__background_compact_find_next_uri(session, uri, next_uri), true);
+        WT_ERR_NOTFOUND_OK(__background_compact_find_next_uri(session, uri, next_uri, &id), true);
```

Comment:
Is the `id` used at all after being set? If not, we could remove it.

Reply:
Good spotting, we no longer use it here.

Reply:
Fixed https://github.com/wiredtiger/wiredtiger/pull/9541/commits/6f54c40e19073acbc673f92af440a2daefa5dee6.